Assets/MixedRealityToolkit/Utilities/StabilizedRay.cs

```diff
@@ -0,0 +1,99 @@
+﻿using UnityEngine;
```

Comment:
Since this contains code that's being moved [out of a file with a Microsoft license header](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7333/files#diff-365ee09e0eb8ac36ceeb5aab05109646L128), one should be added here as well.

Reply:
Fixed